src/test/java/organice/logic/commands/ListCommandTest.java

```diff
@@ -19,21 +22,64 @@
 
     private Model model;
     private Model expectedModel;
+    private Type type;
 
     @BeforeEach
     public void setUp() {
         model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
         expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        type = new Type("patient");
```

Comment:
A suggestion is to declare constants for Type("patient"), Type("donor"), Type("doctor") and use them in your tets.